react/react1/week3/homework/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import './App.css';
+import TodoListTitle from "./TodoListTitle"
+import TimeSpent from "./TimeSpent"
+import TodoItem from "./TodoItem" 
+import BorderComponent from './BorderComponent';
+
+function App() {
+  const [todo, setTodo] = useState([])
+  const [isLoaded, setIsLoaded] = useState(false)
+  const [descriptionState, setDescription] = useState("")
+  const [deadlineState, setDeadline] = useState("")
+  const APIUrl = "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw"
+  const fetchAPI = useCallback(() => {
+    return fetch(APIUrl)
+      .then(response => response.json())
+      .then((todos) => {
+        setTodo(todos, todos.done = false);
+      setIsLoaded(true)
+    })
+  }, [])
+  const descriptionEventHandler = (event) => {
+    setDescription(event.target.value)
+  }
+  const deadlineEventHandler = (event) => {
+    setDeadline(event.target.value)
+  }
+  const addTodo = () => {
+    const newId = todo.length === 0 ? 1 : todo[todo.length-1].id + 1
+    const newTodo = { id: newId, description: descriptionState, deadline: deadlineState, done: false}
+    setTodo(prevTodos => {
+        return [...prevTodos, newTodo]
+    })
+  }
+  useEffect(() => {
+      fetchAPI()
+  }, [fetchAPI])
+  if(isLoaded) return (
+    <div className="App">
+      <BorderComponent color="blue">
```

Comment:
Can you check why this prop is not working? For example, if I change the color from "blue" to "red" I would expect the border color to change to red, but instead, the className changes and appends the red string to the borderComponent. There is no borderComponent-red class defined on the CSS file, so the style breaks and displays nothing when you pass a new prop to the component. Try to improve it and find a solution where the style would not break and actually changes the color.